orange/src/main/java/com/hartwig/hmftools/orange/algo/linx/LinxReportableClusters.java

```diff
@@ -19,9 +20,8 @@ public final class LinxReportableClusters
     @NotNull
     public static Set<Integer> findReportableClusters(@NotNull LinxData linx)
```

Comment:
I think this name is no longer applicable. Simple clusters could be reportable but they are just not visualised? Maybe a better name would be something like `findVisualizedClusters`?